test/test_datasets.py

```diff
@@ -155,6 +160,19 @@ def test_places365_repr_smoke(self):
             dataset = torchvision.datasets.Places365(root, download=True)
             self.assertIsInstance(repr(dataset), str)
 
+    def test_kitti(self):
```

Comment:
We're trying to move away from these tests and instead rely on `ImageDatasetTestCase` as the other classes below in this file. Would you mind porting it there?